build2file

```diff
@@ -21,6 +21,16 @@ libue{spacedungeon} : src/{cxx hxx}{ ** -main -unit-tests/** }
 libue{spacedungeon} : $sfml_libs
 
 exe{spacedungeon} : src/cxx{main} libue{spacedungeon}
+exe{spacedungeon} : data/file{**}
+
+data/file{**} :
+{
+    install = $install.root/data/
+    install.subdirs = true
+}
```

Comment:
Gives error:
error: invalid argument: null value
  build2file:28:28: info: while concatenating dir_path to <untyped>
  info: use quoting to force untyped concatenation

Reply:
> Gives error:
> error: invalid argument: null value
> build2file:28:28: info: while concatenating dir_path to
> info: use quoting to force untyped concatenation

I can't reproduce this error on Windows (can't check on Linux desktop yet) with either `v0.13.0` or the last stage.

However I tried to fix the issue by following the instructions of the error, so please try with that last commit?

Reply:
Checked in a Ubuntu desktop VM and it doesn't work either.
I'll report an issue to the devs.